docs/runtime-contract.md

```diff
@@ -211,9 +211,9 @@ following protocol will be used:
 - `h2c`: HTTP/2 transport, as described in
   [section 3.4 of the HTTP2 spec (Starting HTTP/2 with Prior Knowledge)](https://http2.github.io/http2-spec/#known-http)
 
-Developers SHOULD prefer to use automatic content negotiation where available,
+Developers SHOULD use automatic content negotiation where available,
```

Comment:
'ought' seems appropriate here. 

Also just FYI that we currently do not have automatic content negotiation. See https://github.com/knative/serving/pull/4286

Reply:
used "ought"